JS-Algorithms/challenges.js

```diff
@@ -42,6 +66,16 @@ const COUNTRY_NAMES = ['Germany', 'Norway', 'Island', 'Japan', 'Israel'];
 
 const circularArray = (index) => {
 	// YOUR CODE HERE...
+	const initialArray = COUNTRY_NAMES;
+	let newIndex;
+	if (index > 4) {
+		newIndex = index % 5;
+	} else {
+		newIndex = index;
+	}
+	const firstArray = initialArray.slice(newIndex, 5);
+	const lastArray = initialArray.slice(0, newIndex);
+	return (firstArray.concat(lastArray));
```

Comment:
I see you're using ES6 in other parts of your code, I would encourage using array destructuring instead of `.concat()`